os/board/common/partitions.c

```diff
@@ -92,6 +94,7 @@ struct partition_data_s g_flash_part_data = {
 
 #ifdef CONFIG_SECOND_FLASH_PARTITION
 struct partition_data_s g_second_flash_part_data = {
+	CONFIG_SECOND_FLASH_START_ADDR,
```

Comment:
#ifndef CONFIG_SECOND_FLASH_START_ADDR
#define CONFIG_SECOND_FLASH_START_ADDR 0
#endif

Reply:
signing verification function has a parameter of actual flash address. So it should be set as actual address.
So I added #error condition to make build break.